### ISSUE TITLE
Bomberman 64 graphical errors

### ISSUE THREAD

**Issue by carnage702** [emukidid/Wii64#5]
when you start bomberman 64 there will be alot of stuff missing and you hit A and start  until you reach ingame  after that just restart the rom after  it made an autosave wait until the intro shows a picture and you can now see the world map and go ingame while you see the actual game, also when passing under some objects the vertexes will stretch and fill the screen, this happens on glide and rice.

**Comment by emukidid** [comment]
When reporting these, can you check against the PC version of the plugins as well?

**Comment by carnage702** [comment]
wich plugin exactly do you want me to test? any specific version?

**Comment by emukidid** [comment]
glN64 0.41 (what we started the port on) and the latest version of Rice you can find out there.

**Comment by carnage702** [comment]
you are correct bomberman 64 has severe issues on muppen 64 with glN64 0.41 or rice 6.1.4, i cant even get ingame on that so if the issue is the same as on the muppen 64 emulator i should not report it from this moment on?

**Comment by emukidid** [comment]
You can report it but it will help greatly to know whether the issue lies in the work done porting it to GC/Wii or if it's a problem for the original plugin too. (If it's the latter case it's less likely to be fixed really)

**Comment by emukidid** [comment]
This is fixed in 1.4.4 in glN64.
